docs/coding-guidelines/project-guidelines.md

```diff
@@ -161,6 +162,9 @@ Tests don't need to reference default references which are part of the targeting
 ### tests output
 All test outputs should be under
 
+## gen
+In the gen directory any source generator related to the assembly should exist. This does not mean the source generator is only used for that assembly only that it is conceptually apart of that assembly. For example, the assembly may provide attributes or low-level types the source generator uses.
+
 `bin\$(MSBuildProjectName)\$(TargetFramework)`
```

Comment:
Was this meant to be part of tests output paragraph? This looks out-of-place now.

Reply:
Doh.